internal/ignition/disconnected.go

```diff
@@ -172,6 +198,14 @@ func createManifests(infraEnv *common.InfraEnv, disconnectedDir string) error {
 		return errors.Wrap(err, "failed to create pull secret manifest")
 	}
 
+	if err := createNMStateConfigManifests(infraEnv, manifestsDir, log); err != nil {
+		return errors.Wrap(err, "failed to create NMStateConfig manifests")
+	}
+
+	if err := createAgentConfigManifest(cluster, manifestsDir, log); err != nil {
```

Comment:
info: agent-config.yaml does not need to be present in the cluster-manifests dir like infraenv.yaml